bugbot/rules/component.py

```diff
@@ -89,6 +89,55 @@ def get_bugs(self, date="today", bug_ids=[]):
         # Classify those bugs
         bugs = get_bug_ids_classification("component", bug_ids)
 
+        # Collect bugs classified as Fenix::General
+        fenix_general_bug_ids = [
+            bug_id
+            for bug_id, bug_data in bugs.items()
+            if bug_data.get("class") == "Fenix"
+            and bug_data["prob"][bug_data["index"]]
+            >= self.get_config("general_confidence_threshold")
+        ]
+
+        def get_confidence_threshold(
+            bug_data, general_confidence_threshold, confidence_threshold
+        ):
+            if bug_data["class"] == "General":
+                return general_confidence_threshold
+            return confidence_threshold
+
+        # Collection bugs that were originally Fenix::General but reclassified to another product with low confidence
+        originally_fenix_general_bug_ids = [
+            bug_id
+            for bug_id, bug_data in bugs.items()
+            if raw_bugs[bug_id]["product"] == "Fenix"
+            and raw_bugs[bug_id]["component"] == "General"
+            and bug_data["prob"][bug_data["index"]]
+            <= get_confidence_threshold(
+                bug_data,
+                self.get_config("general_confidence_threshold"),
```

Comment:
We could store all thresholds in variables at the top and reuse the variables instead of repeatedly calling `self.get_config()`.

Reply:
Stored all the thresholds as variables in the initialization: bc68d06132884591efb2653977567238bd0ea681

The rest of `get_bugs()` still uses `self.get_config()` but it may be out of this PR's scope. WDYT?

Reply:
Yes, we could do it later in a separate PR. Feel free to file a follow-up issue.

Reply:
I'll be sure to file an issue once this PR is merged!

Reply:
You could file the issue before merging the PR if you want.

Reply:
Done here https://github.com/mozilla/bugbot/issues/2439